Use unique names for commonly used parameters

I tried building a project referencing Microsoft.AspNetCore.All + Razor.Design, and got this error:

```
\.nuget\packages\microsoft.aspnetcore.razor.design\2.1.0-preview1-28042\build\netstandard2.0\Microsoft.AspNetCore.Razor.Design.targets(127,5): error MSB4062: The "Microsoft.AspNetCore.Razor.Tasks.RazorGenerate" task could not be loaded from the assembly C:\Users\Pranav\.nuget\packages\microsoft.extensions.fileproviders.embedded\2.1.0-preview1-28042\build\netstandard2.0\..\..\tasks\netstandard1.5\Microsoft.Extensions.FileProviders.Embedded.Manifest.Task.dll
```

This is because the two packages share the same property names - https://github.com/aspnet/FileSystem/blob/dev/src/FS.Embedded/build/netstandard2.0/Microsoft.Extensions.FileProviders.Embedded.targets#L4. We need to scan the targets and ensure we're using unique names.

--- a/src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props
+++ b/src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props
@@ -1,4 +1,4 @@
-﻿<Project TreatAsLocalProperty="TaskFolder;TaskAssembly">
+﻿<Project TreatAsLocalProperty="_RazorTaskFolder;_RazorTaskAssembly">
   <!-- 
     Properties and tasks supporting Razor MSBuild integration
   -->
@@ -22,10 +22,10 @@
     <!-- Override this to hijack the tasks and targets. Used by tests. -->
     <_RazorMSBuildRoot Condition="'$(_RazorMSBuildRoot)'==''">$(MSBuildThisFileDirectory)..\..\</_RazorMSBuildRoot>
     
-    <TaskFolder Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netstandard2.0</TaskFolder>
-    <TaskFolder Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net46</TaskFolder>
+    <_RazorTaskFolder Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netstandard2.0</TaskFolder>
+    <_RazorTaskFolder Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net46</TaskFolder>
     
-    <_RazorTaskAssembly Condition="'$(_RazorTaskAssembly)'==''">$(_RazorMSBuildRoot)\tasks\$(TaskFolder)\Microsoft.AspNetCore.Razor.Tasks.dll</_RazorTaskAssembly>
+    <_RazorTaskAssembly Condition="'$(_RazorTaskAssembly)'==''">$(_RazorMSBuildRoot)\tasks\$(_RazorTaskAssembly)\Microsoft.AspNetCore.Razor.Tasks.dll</_RazorTaskAssembly>
   </PropertyGroup>
 
   <UsingTask TaskName="Microsoft.AspNetCore.Razor.Tasks.RazorGenerate" AssemblyFile="$(_RazorTaskAssembly)" />

